### PR TITLE
275 updating error copies on frontend

### PR DESCRIPTION
**Pull request by prakash-bhatt-rs**
#275

### CODE REVIEW

File: ssr/src/page/err.rs [hunk 1]
```diff
@@ -34,22 +43,25 @@ pub fn ErrorView(#[prop(into)] error: MaybeSignal<String>) -> impl IntoView {
...
+            <h1 class="p-2 text-2xl md:text-3xl font-bold text-white">"oh no!"</h1>
+            <div class="text-center text-xs md:text-sm text-white/60 w-full md:w-2/3 lg:w-1/3 resize-none px-8 mb-4">
+                {error.clone()}
+            </div>
+            // <textarea
+            // prop:value=error
+            // disabled
+            // rows=3
+            // class="text-center bg-white/10 text-xs md:text-sm text-white/60 w-full md:w-2/3 lg:w-1/3 resize-none p-2 mb-4"
+            // ></textarea>
```
Comment (ravibazz) [inline ssr/src/page/err.rs:59]: Do we need this commented code?

**Comment by ravibazz** [comment]
@harshitasrivastav28 is this tested on staging?

File: ssr/src/page/err.rs (hunk 1 above)
Comment (prakash-bhatt-rs) [reply ssr/src/page/err.rs:59]: No, we can remove it @ravibazz

**Comment by prakash-bhatt-rs** [comment]
@ravibazz final UI:
Screenshot 2024-08-06 140002

**Review by ravibazz — APPROVED** [review]

**Pull request merged**